base/uk.ac.stfc.isis.ibex.ui.synoptic/src/uk/ac/stfc/isis/ibex/ui/synoptic/SynopticSelectionDialog.java

```diff
@@ -53,22 +53,32 @@ public SynopticSelectionDialog(
 		super(parentShell, title);
 		this.available = available;
 	}
-	
+
 	/**
 	 * Get the synoptic that the user has selected.
 	 * @return The synoptic that the user has selected.
 	 */
 	public SynopticInfo selectedSynoptic() {
 		return selectedSynoptic;
 	}
-	
+
 	@Override
 	protected void okPressed() {
         selectedSynoptic = SynopticInfo.search(available, items.getSelection()[0].getText());
 		
 		super.okPressed();
 	}
-	
+
```

Comment:
You add this function in two places. Can if live in a utility function file and be referenced?